src/code/audio_seqplayer.c

```diff
@@ -1202,7 +1202,7 @@ void AudioSeq_SequenceChannelProcessScript(SequenceChannel* channel) {
                     case 0xCB:
                         offset = (u16)parameters[0];
 
-                        scriptState->value = *(seqPlayer->seqData + (u32)(offset + scriptState->value));
+                        scriptState->value = *(seqPlayer->seqData + (uintptr_t)(offset + scriptState->value));
```

Comment:
offset is u16, is being added to a u8.  there is no need to cast this up from u32 to uintptr_t